src/components/Frame/Frame.tsx

```diff
@@ -376,10 +376,8 @@ class Frame extends React.PureComponent<CombinedProps, State> {
   };
 
   private handleClick = () => {
-    if (this.skipToMainContentTargetNode.current == null) {
-      return;
-    }
-    this.skipToMainContentTargetNode.current.focus();
+    this.skipToMainContentTargetNode.current &&
```

Comment:
`skipToMainContentTargetNode ` will never be null after mount, this is just a shortcut and will give us coverage